apps/src/studio/studio.js

```diff
@@ -3934,10 +3970,35 @@ Studio.setSprite = function (opts) {
     spriteWalk.setAttribute('height', sprite.drawHeight * sprite.frameCounts.walk); // 1200
   }
 
+  // Set up movement audio for the selected sprite (should be preloaded)
+  Studio.movementAudioOptions = Studio.movementAudioOptions || {};
```

Comment:
This initialization pattern is a little different from what we typically use. I might prefer to see us explicitly initialize `Studio.movementAudioOptions = {}` in Studio.init(). Partly for consistency and clarity, and partly because I've been burned before with mochaTest where it runs multiple levels in sequence on the same `Studio` instance. There could be an issue there where the skin changed between levels, but we're partially initialized based on the previous level. I'm not thrilled that our tests do this, but it does happen today - and we work around problems like this with test code that attempts to patch up state between levels. (although eventually, we should change Studio away from being a global)